contracts/governance/mocks/MockTimeLock.sol

```diff
@@ -10,12 +11,14 @@ contract MockTimeLock is Timelock {
      * @param delay_ The timestamp of delay for timelock contract
      */
     function mockInitialize(address admin_, uint256 delay_) external {
-        require(!initalized, "Already initialized");
+        Claimable.initialize(msg.sender);
         admin = admin_;
         pauser = admin_;
         delay = delay_;
+    }
 
-        owner_ = msg.sender;
-        initalized = true;
+    // test function to set delay
+    function setDelayMock(uint256 _delay) external {
```

Comment:
Could you flesh out the doc string?

Also nit: maybe we can call this mockSetDelay() for consistency